### ISSUE TITLE
Should read image's alt text?

### ISSUE THREAD

**Issue by ken107** [ken107/read-aloud#143]

**Comment by ken107** [comment]
https://stackoverflow.com/questions/58447538/accessibility-difference-between-img-alt-and-figcaption

We will not implement it.  For a figure, the figcaption element should be used instead.  For a functional image (e.g. button image), we're not really a screen reader, as we're not reading links and buttons and navigations.
